vm/src/vm/setting.rs

```diff
@@ -74,6 +74,8 @@ pub struct Settings {
     /// false for wasm. Not a command-line option
     pub allow_external_library: bool,
 
+    pub utf8_mode: bool,
```

Comment:
I have no idea about this variable, but keeping it as same type as flag looks reasonable. So probably `u8` here too?

Reply:
Yeah, I think that'd make sense.

Reply:
Done!